.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
same with this file

Reply:
That should not be there, it got missed.